.github/workflows/release.yml

```diff
@@ -0,0 +1,57 @@
+name: Release
+
+permissions:
+  contents: write
+
+on:
+  push:
+    tags:
+      - mountpoint-s3-[0-9]+.*
+      - mountpoint-s3-[a-z]+-[0-9]+.*
+
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: taiki-e/create-gh-release-action@v1
+      with:
+        prefix: mountpoint-s3(-[a-z]+)?
+        draft: true
+        title: "$prefix $version"
```

Comment:
Let's use a nicer title: "Mountpoint for Amazon S3 $version"

Reply:
changed